website/Gemfile

```diff
@@ -0,0 +1,5 @@
+source "https://rubygems.org"
+gem "github-pages", "~> 227"
+gem "rake", "~> 13.0" # Required for "just-the-docs" theme.
```

Comment:
If we manage just-the-docs in this Gemfile, then we no longer need to specify `rake` :smile: